lib/commands/search.jsx

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const { h, render } = require('ink')
+const { Search } = require('../components/search.jsx')
+
+const SearchCommand = module.exports = {
+  command: 'search',
+  aliases: ['s', 'se', 'find'],
+  describe: 'Search the NPM registry',
+  builder (y) {
+    return y.help().alias('help', 'h')
+      .options(SearchCommand.options)
+  },
+  options: Object.assign(require('../common-opts.js'), {}),
```

Comment:
@zkat [What you were saying over here](https://npm.community/t/tink-new-search-ui/3087/22?u=chrisforrette) makes sense about passing the whole options object to `libnpm.search`—so just to confirm, I don't need `common-opts.js` at all for the search command, right? So I can change this to a `figgy-pudding` object with [all the `libnpmsearch` opts](https://www.npmjs.com/package/libnpmsearch#opts-for-libnpmsearch-commands)?